frontend/src/components/dashboard/GalleryPreview.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import Typography from '@material-ui/core/Typography';
+import ImgMediaCard from "./ImgMediaCard";
+
+const useStyles = makeStyles((theme) => ({
+  galleryTitle: {
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'flex-start',
+    padding: theme.spacing(5)
+  },
+  viewMoreButton: {
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'flex-end',
+    padding: theme.spacing(4)
+  },
+  link: {
+    textDecoration: 'none',
+    color: theme.palette.text.primary
+  }
+}));
+
+export default function GalleryPreview(props) {
+  const classes = useStyles();
+
+  return (
+    <>
+      <Container className={classes.galleryTitle}>
+        <Typography variant="h4">{props.name}</Typography>
+      </Container>
+      <Grid
+        container
+        direction="row"
+        justify="center"
+        alignItems="center"
```

Comment:
I think since this is technically a JS file, it is best practice to use ' instead of "

Reply:
It looks like React is a little different: https://airbnb.io/javascript/react/#quotes
Seems like double quotes for jsx elements but single quotes for things like imports